domains/hashir.is-not-a.dev.json

```diff
@@ -0,0 +1,14 @@
+{
+    "domain": "is-not-a.dev",
+    "subdomain": "hashir",
+
+    "owner": {
+        "email": "hmserofficial@gmail.com"
+    },
+
+    "record": {
+        "CNAME": "hmser.netlify.app"
```

Comment:
![image](https://github.com/open-domains/register/assets/76603072/71f60ed2-b1cd-43ff-bc20-14dcb47252fb)
We don't support netlify.